### ISSUE TITLE
macOS rancher desktop cannot work if /private/etc/sudoers.d does not exists

### ISSUE THREAD

**Issue by nolith** [rancher-sandbox/rancher-desktop#4316]
### Actual Behavior

On a fresh install, if `/private/etc/sudoers.d` does not exists, rancher desktop fails to create `/private/etc/sudoers.d/zzzzz-rancher-desktop-lima` and does not start properly.

### Steps to Reproduce

A clean macOS does not have the `/private/etc/sudoers.d` folder, it can be removed with `sudo rm -rf /private/etc/sudoers.d/` ⚠️ make sure there are no other files in the folder

open rancher desktop

![Screenshot 2023-03-29 at 23 03 26](https://user-images.githubusercontent.com/78752/228667498-313c5abe-d0e3-443e-a914-878fa3de7eb2.png)

click "ok"

enter the admin password

see the failure

### Result

```
Error: Command failed: /bin/sh -xec 'cp "/var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt" /private/etc/sudoers.d/zzzzz-rancher-desktop-lima && rm -f "/var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt"'
+ cp /var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt /private/etc/sudoers.d/zzzzz-rancher-desktop-lima
cp: /private/etc/sudoers.d/zzzzz-rancher-desktop-lima: No such file or directory
```

From the logs:

```
2023-03-29T21:03:37.316Z: Error starting lima: Error: Command failed: /bin/sh -xec 'cp "/var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt" /private/etc/sudoers.d/zzzzz-rancher-desktop-lima && rm -f "/var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt"'
+ cp /var/folders/sj/6sy5qhz51qxcyy2kx_plq9jr0000gn/T/rd-sudoers00699054.txt /private/etc/sudoers.d/zzzzz-rancher-desktop-lima
cp: /private/etc/sudoers.d/zzzzz-rancher-desktop-lima: No such file or directory

    at /Applications/Rancher Desktop.app/Contents/Resources/app.asar/node_modules/sudo-prompt/index.js:390:27
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
  code: 1
```

### Expected Behavior

No errors and a working rancher desktop

### Rancher Desktop Version

1.8.1

### Rancher Desktop K8s Version

1.25.7

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Ventura 13.3 (22E252)

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by nolith** [comment]
Running `sudo mkdir /private/etc/sudoers.d` and restarting rancher desktop fixes this problem

**Comment by jandubois** [comment]
@nolith Thank you for reporting this. I thought `/etc/sudoers.d` is created as part of the OS installation. Do you happen to know why it does not exist on your machine? Is it removed by enterprise management software, or am I wrong, and a fresh install doesn't include this directory anymore?

**Comment by nolith** [comment]
@jandubois I actually have no idea. This is a private machine without any enterprise management software.

Today I just uninstalled docker desktop and installed rancher desktop and this is what happened.

Out of curiosity I made this simple patch https://github.com/rancher-sandbox/rancher-desktop/commit/2df2cf3be8b32673fafe7832b19931fbde8c5029, I'm not a typescript developer, but if you think it makes sense I can make a PR.
